examples/main/README.md

```diff
@@ -208,6 +208,14 @@ Top-p sampling, also known as nucleus sampling, is another text generation metho
 
 Example usage: `--top-p 0.95`
 
+### Min P Sampling
+
+-   `--min-p N`: Sets a minimum base probability threshold for token selection (default: 0.0).
+
+Min-P sampling is a sampling method where the value represents the base required probability percentage. This value is scaled down based on the probability of the most likely token. For example, with a Min-P value set at 0.05 and the highest token probability being 90%, the percentage requirement threshold becomes 4.5%. This approach ensures a balance of quality and variety in the results, and was designed as an alternative to improve upon Top P. The default value is 0.05.
```

Comment:
Here's how I would word it:
```suggestion
The Min-P sampling method was designed as an alternative to Top-P, and aims to ensure a balance of quality and variety. The parameter *p* represents the minimum probability for a token to be considered, relative to the probability of the most likely token. For example, with *p*=0.05 and the most likely token having a probability of 0.9, logits with a value less than 0.045 are filtered out.
```

Reply:
I like this one way better. @kalomaze

Reply:
Well written, I'll go with this once I get word back on whether or not it should be on by default.

Reply:
Yeah, I prefer the raw decimal values over percentages, since it maps onto how the values are used overall.